api/proto/teleport/legacy/types/types.proto

```diff
@@ -6063,3 +6063,17 @@ message GCPMatcher {
   // ProjectIDs are the GCP project ID where the resources are deployed.
   repeated string ProjectIDs = 4 [(gogoproto.jsontag) = "project_ids,omitempty"];
 }
+
+// KubernetesMatcher matches Kubernetes services.
+message KubernetesMatcher {
```

Comment:
Do we need to add this to our legacy protos or could we move it to non-legacy files?

Reply:
For now all matchers live in the legacy protos. We'll need separate PR to move them all out, I'll need to sync with Marco about this.